R/ParseUserEnteredTable.R

```diff
@@ -168,11 +168,23 @@ parseAsVectorOrMatrix <- function(m, warn = FALSE)
     if (any(!isTextNumeric(m[-1,-1], allow.missing = TRUE)))
         row.names.given <- isQStatistic(m[1,1])
 
+    # If input data are percentages then we can do more tests
+    .is_pct <- function(x) { tmp <- asNumericVector(x); return(!is.null(attr(tmp, "statistic"))) }
+
     # Handling ambiguous row/column labels
     if (is.na(row.names.given) || is.na(col.names.given))
     {
+        # If input data are percentages then we can do more tests
+        .is_pct <- function(x) { tmp <- asNumericVector(x); return(!is.null(attr(tmp, "statistic"))) }
```

Comment:
Does this need to be defined again?

Reply:
Perhaps the first `.is_pct` definition can be removed

Reply:
Thanks for picking that up. Removed the first one in eb80adb